website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+<!-- TODO: add more examples -->
+
+#### Visual organization
+
+Once a logical organization or grouping has been established,
+
+- categorize elements and fields into sections, 
+- introduce typographic elements to establish hierarchy,
+- and if necessary use dividers to more clearly differentiate sections.
+
+Each one of these methods will visually help the user to better parse and understand the relationships between each section and fields contained within.
+
+## Sizing
+
+### Length
+
+Determined by the number of fields and supporting elements (text, titles, etc). form length can have a significant impact on whether a user successfully completes a form or abandons the task prior to completion. If a feature or flow relies on a large number of fields, consider breaking up the form using one or more of the following strategies.
+
+#### Multiple sections
+
+Breaking up a form into multiple sections based on the relationship between fields and categories of sections can make a long form seem less complex. Introducing typographic elements and dividers can further aid the hierarchy of the form and differentiate sections from one-another.
+
+#### Multiple steps or pages
+
+For exceedingly long, complex forms (e.g., creating a cluster), consider breaking the form into multiple steps or pages. A [Stepper](/components/stepper-indicator) can be used to annotate status and location in a multi-step form.
+
+!!! Info
+
+As a general note, special care should be taken to _reduce_ the number of fields whenever possible.
+
+- Long forms are more challenging for the user which can result in a lower completion rate.
+- If a field is optional, consider whether it should be included in the form at all. If the information isn't crucial to the experience or feature, why is it being collected at all?
+!!!
+
+#### Using dividers
+
+Dividers introduce more visual hierarchy and differentiation in long, complex forms.
+
+Only use dividers between sections of a form, not between fields or other smaller elements within a section. A divider should be used to break up different types of content and categories within a form.
+
+When using a divider, include a 24px gap above and below to separate the divider from surrounding form elements, otherwise it can appear "attached" to a specific section or field.
+
+![Spacing using dividers](/assets/patterns/form-patterns/divider-spacing.png =500x*)
+
+!!! Dont
+
+Dividers should not be used trailing the last section of a form between the fields and the button set or actions. Instead, the section spacing of 32px should be used.
+!!!
+
+## Width
+
+The width of a form and the fields it contains is largely dependent on the context the form is in, but adhering to these high-level guidelines can make a complex form approachable and introduce consistency in the UX.
+
+#### Form width
+
+A form should hug the content and the fields it contains.
+
+![Form width](/assets/patterns/form-patterns/form-width-example.png)
```

Comment:
Can we provide more realistic examples? Plopping a radio card example within a credit card form feels a little weird.